charts/headlamp/templates/clusterrolebinding.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cluster-admin
+  name: {{ .Values.clusterRoleBinding.clusterRoleName | default "cluster-admin" }}
```

Comment:
How about we just have

```suggestion
  name: {{ .Values.clusterRoleBinding.clusterRoleName
```

and that default value should be in `values.yaml`.

Reply:
yes, sure